README.rst

```diff
@@ -110,9 +110,7 @@ Pre-trained models
 |            |                                                                                              | Mozilla Common Voice  |
 |            |                                                                                              | (en_1488_2019-12-10), |
```

Comment:
This should also be fixed--the published checkpoint was trained on the version from June 2019.

Reply:
yes, looks like there are multiple changes that need to be made, let me continue working on this